tests/e2e/__testers__.ts

```diff
@@ -75,6 +86,20 @@ export const testTemplate = (params: {
     expect(stripAnsi(initResult.stdout)).toMatch('Generated Prisma Client')
     expect(stripAnsi(initResult.stdout)).toMatch('The seed command has been executed.')
 
+    // TODO only empty template does not have this artifact. Our short circuit above should narrow the type here...
+    if ('prisma/seed.js' in ctx.template.artifacts) {
+      const prisma = await ctx.getPrisma()
+      try {
+        const seedModuleFile = ctx.template.artifacts['prisma/seed.js']
+        ctx.fs.write('seedModule.js', seedModuleFile.content)
```

Comment:
We do not transform the seed module into an eval-able function. Doing so would require a bit more consideration:

1. Eval doesn't work with async? But Node VM module might which anyways we should prefer?
2. Need to turn imports into requires, lots of edge-cases there...